app/build.gradle

```diff
@@ -85,12 +85,21 @@ dependencies {
     implementation "net.e175.klaus:solarpositioning:0.0.9"
     // use pristine greendao instead of our custom version, since our custom jitpack-packaged
     // version contains way too much and our custom patches are in the generator only.
+
+    implementation 'com.twofortyfouram:android-plugin-client-sdk-for-locale:[4.0.3, 5.0['
+    implementation 'com.twofortyfouram:android-plugin-host-sdk-for-locale:[2.0.3,3.0['
+    implementation 'com.twofortyfouram:android-plugin-api-for-locale:[1.0.2,2.0['
+
     implementation "org.greenrobot:greendao:2.2.1"
     implementation "org.apache.commons:commons-lang3:3.7"
     implementation "org.cyanogenmod:platform.sdk:6.0"
     implementation 'com.jaredrummler:colorpicker:1.0.2'
 //    implementation project(":DaoCore")
     implementation 'com.github.wax911:android-emojify:0.1.7'
+
+    implementation 'com.android.support:support-v4:28.1.0'
+
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
Why is that needed?

Reply:
We already have 
com.android.support.constraint:constraint-layout:1.1.3 in .... maybe just a duplicate, but not critical